src/org/opensolaris/opengrok/configuration/messages/Response.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * CDDL HEADER START
+ *
+ * The contents of this file are subject to the terms of the
+ * Common Development and Distribution License (the "License").
+ * You may not use this file except in compliance with the License.
+ *
+ * See LICENSE.txt included in this distribution for the specific
+ * language governing permissions and limitations under the License.
+ *
+ * When distributing Covered Code, include this CDDL HEADER in each
+ * file and include the License file at LICENSE.txt.
+ * If applicable, add the following below this CDDL HEADER, with the
+ * fields enclosed by brackets "[]" replaced with your own identifying
+ * information: Portions Copyright [yyyy] [name of copyright owner]
+ *
+ * CDDL HEADER END
+ */
+
+/*
+ * Copyright (c) 2018 Oracle and/or its affiliates. All rights reserved.
+ */
+package org.opensolaris.opengrok.configuration.messages;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class Response {
```

Comment:
add comment pls about what it is and how it is used

Reply:
I'm not a big fun of comments for no library-like code. Comment like 
```java
/**
 * Generic message response.
 */
```
is useless because it can be immediately seen from the package and the class name.

However, if I were to write
```java
/**
 * Generic message response. It allows to combine multiple String responses into one because...
 */
```
then I'm going into details which are out of scope of the class.

Reply:
but the purpose for the opengrok and the usage in the opengrok is not self-evident

Reply:
Could you please help me out? I'm still not sure what exactly is missing because for me as an author it is self-evident.

Reply:
And I'm not kidding this happened to me - I saw a Response and I had to go and search where it's used, what kind of data it transports and what to expect from it when looking into this PR.